modules/imgproc/perf/perf_cvt_color.cpp

```diff
@@ -263,6 +263,11 @@ PERF_TEST_P(Size_CvtMode, cvtColor8u,
 
 #if defined(__APPLE__) && defined(HAVE_IPP)
     SANITY_CHECK(dst, _mode == CX_BGRA2HLS_FULL ? 2 : 1);
+#elif defined(_MSC_VER) && _MSC_VER >= 1900 /* MSVC 14 */
+    if (_mode == CX_Luv2BGRA)
+        SANITY_CHECK_NOTHING();
+    else
+        SANITY_CHECK(dst, 1);
```

Comment:
Executable copied from other machine (VS2013) passes, but the one built with VS2015 fails:
```
[ RUN      ] Size_CvtMode_cvtColor8u.cvtColor8u/79
C:\builds\precommit_windows_ten\opencv\modules\ts\src\ts_perf.cpp(398): error: The difference between expect_rng2 and actual_rng2 is 255, which exceeds eps, where
expect_rng2 evaluates to 0,
actual_rng2 evaluates to 255, and
eps evaluates to 1.
Argument "dst" has unexpected value of the [72:57:2] element

params    = (127x61, CX_Luv2BGRA)
termination reason:  reached maximum number of iterations
bytesIn   =      23241
bytesOut  =      30988
samples   =          1
outliers  =          0
frequency =    3220800
min       =       1026 = 0.32ms
median    =       1026 = 0.32ms
gmean     =       1026 = 0.32ms
gstddev   = 0.00000000 = 0.00ms for 97% dispersion interval
mean      =       1026 = 0.32ms
stddev    =          0 = 0.00ms
[  FAILED  ] Size_CvtMode_cvtColor8u.cvtColor8u/79, where GetParam() = (127x61, CX_Luv2BGRA) (33 ms)
```